### ISSUE TITLE
[ED][Project Plan] "WCAG Checkpoints"

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-roles-responsbilities#7]
Consider using a different term. [WCAG 1.0 had "checkpoints"](https://www.w3.org/TR/WCAG10/checkpoint-list.html) (where as 2 has "success criteria") and so it could be quite confusing for old timers -- that is "WCAG Checkpoints" to some people still means either "WCAG 1.0 Checkpoints" or WCAG 2 success criteria. ;-)

**Comment by orvalle** [comment]
Our use of checkpoints should be restricted to checkpoints that are part of a checklist used to evaluate WCAG 2.0. I don't think we're using the 1.0 definition or actual checkpoints in any way.

**Comment by dboudreau** [comment]
@slhenry - I feel like WCAG 1.0 checkpoints are antediluvian terms that nobody excepts us actually refers to anymore, but I see your point. We can think about using a different term, no problem. We'll give it some thought and make changes accordingly.

**Comment by dboudreau** [comment]
After having spoken to the editing team, we feel that there is not a lot of risk for confusion here, as the checkpoints document itself will not be shared publicly and the reference to the considerations / checkpoints will be the list of elements to keep in mind to define what each Success Criterion means to a particular role. To address your concern, I've changed the wording to "Complete set of accessibility considerations derived from WCAG". Also changed "checkpoint" to "consideration" whenever appropriate.

If there is still concern about the label when we get to that, we will change it. But for the time being, we feel it's more productive to move forward and cross that bridge when we get to it. Please close if you agree.

**Comment by shawna-slh** [comment]
Cool.
